packages/server/src/routes/me/index.ts

```diff
@@ -64,17 +70,33 @@ export const sendInvite = (
       throw new MissingParamsError(["inviteEmail"]);
     }
 
+    if (videoToken && isJointVideo === undefined) {
+      throw new MissingParamsError(["isJointVideo"]);
+    }
+
+    // TODO generate this server side somewhere.
+    const inviteToken = videoToken;
+
+    if (inviteToken) {
+      const newInvite: OperationToInsert = {
+        creator_uid: loggedInMemberId,
+        op_code: OperationType.INVITE,
+        created_at: firestore.FieldValue.serverTimestamp(),
+        data: {
+          invite_token: inviteToken,
+          is_joint_video: !!isJointVideo,
+          video_token: inviteToken
```

Comment:
Maybe we should just store a video url rather than a token, to be really explicit—right now the way the video token gets converted to an actual video url is implicit. Alternatively, if `videoToken` is better for some reason, then we can just have a function somewhere that does the conversion/if that might need to be done elsewhere, that it's exposed through `shared` or `client`; though that seems unlikely